### ISSUE TITLE
LoadLibrary failed with error 126

### ISSUE THREAD

**Issue by weiyouren** [dme-compunet/YoloV8#85]
My environment is win10 x64 cuda11.8 and cudnn 8, my environment is good because I have no problem training and predit with YOLOV8
and i test python onnx with 1.19.2
```python
import onnxruntime as ort
print(ort.get_device())  # output GPU
```
when I export onnx：
```python
from ultralytics import YOLO

# Load a model
model = YOLO(r'xxx')

# Export the model to ONNX format
model.export(format='onnx')
```
output is:
Ultralytics YOLOv8.2.70 🚀 Python-3.10.9 torch-2.4.0+cu118 CPU (Intel Xeon Silver 4110 2.10GHz)
Model summary (fused): 168 layers, 11,125,971 parameters, 0 gradients, 28.4 GFLOPs

PyTorch: starting from 'F:\xxx.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 5, 8400) (21.5 MB)

ONNX: starting export with onnx 1.17.0 opset 19...
ONNX: export success ✅ 3.7s, saved as 'F:\xxx.onnx' (42.6 MB)

Export complete (12.5s)

So I need to use onnx runtime 1.17 instead of 1.19?

now i use nuget yolov8.gpu v5.1 error:
[ErrorCode:RuntimeException] D:\\a\\_work\\1\\s\\onnxruntime\\core\\session\\provider_bridge_ort.cc:1637 onnxruntime::ProviderLibrary::Get [ONNXRuntimeError] : 1 : FAIL : LoadLibrary failed with error 126 \"\" when trying to load \"xxx\\runtimes\\win-x64\\native\\onnxruntime_providers_cuda.dll\"\n

or i use nuget yolov8 v5.1 with cpu error:
System.InvalidOperationException: **The metadata parsing failed, making sure you use an official YOLOv8 model**
 ---> System.NotSupportedException: Unrecognized YOLO model architecture
   at Compunet.YoloV8.Metadata.YoloMetadata.GetYoloArchitecture(String description)
   at Compunet.YoloV8.Metadata.YoloMetadata..ctor(Dictionary`2 metadata)
   at Compunet.YoloV8.Metadata.YoloMetadata..ctor(InferenceSession session)
   at Compunet.YoloV8.Metadata.YoloMetadata.Parse(InferenceSession session)
   --- End of inner exception stack trace ---

So what should I do

**Comment by dme-compunet** [comment]
Please check version 5.1.1

**Comment by weiyouren** [comment]
version 5.1.1 same issue
this is my model, you can try it out
[test.zip](https://github.com/user-attachments/files/17388029/test.zip)
